.circleci/docker/Dockerfile.ubuntu1904

```diff
@@ -34,7 +36,7 @@ RUN set -ex; \
 		libboost-regex-dev libboost-filesystem-dev libboost-test-dev libboost-system-dev \
 		libboost-program-options-dev \
 		libjsoncpp-dev \
-		llvm-8-dev libcvc4-dev libleveldb1d \
+		llvm-8-dev libcvc4-dev libz3-static-dev libleveldb1d \
```

Comment:
If the static Z3 build turns out to work, we might want to remove libcvc4-dev from the image.